wizard/WizardCreateDevice1.qml

```diff
@@ -185,12 +185,7 @@ Rectangle {
                         wizardController.walletOptionsSubaddressLookahead = lookahead.text;
                     var _restoreHeight = 0;
                     if(restoreHeight.text){
-                        // Parse date string or restore height as integer
-                        if(restoreHeight.text.indexOf('-') === 4 && restoreHeight.text.length === 10){
-                            _restoreHeight = Wizard.getApproximateBlockchainHeight(restoreHeight.text, Utils.netTypeToString());
-                        } else {
-                            _restoreHeight = parseInt(restoreHeight.text)
-                        }
+                        _restoreHeight = Utils.parseDateStringOrRestoreHeightAsInteger(restoreHeight.text);
```

Comment:
```
wizardController.walletOptionsRestoreHeight = Utils.parseDateStringOrRestoreHeightAsInteger(restoreHeight.text);
```

Just set it directly, no need for an extra variable.

Same in wizard/WizardRestoreWallet1.qml.